shuup/admin/modules/contact_group_price_display/views/list.py

```diff
@@ -0,0 +1,60 @@
+# -*- coding: utf-8 -*-
+# This file is part of Shuup.
+#
+# Copyright (c) 2012-2018, Shuup Inc. All rights reserved.
+#
+# This source code is licensed under the OSL-3.0 license found in the
+# LICENSE file in the root directory of this source tree.
+from __future__ import unicode_literals
+
+from django.utils.encoding import force_text
+from django.utils.translation import ugettext_lazy as _
+
+from shuup.admin.modules.contact_group_price_display.views.forms import (
+    get_price_display_mode, PRICE_DISPLAY_MODE_CHOICES
+)
+from shuup.admin.shop_provider import get_shop
+from shuup.admin.toolbar import NewActionButton, SettingsActionButton, Toolbar
+from shuup.admin.utils.picotable import Column
+from shuup.admin.utils.views import PicotableListView
+from shuup.core.models._contacts import (
+    ContactGroupPriceDisplay, get_groups_for_price_display_create,
+    get_price_displays_for_shop
+)
+
+
+class ContactGroupPriceDisplayListView(PicotableListView):
+    model = ContactGroupPriceDisplay
+    default_columns = [
+        Column("group", _(u"Group"), display="group"),
+        Column("display_mode", _(u"Display Mode"), display="show_display_mode")
+    ]
+
+    def get_queryset(self):
+        if self.request.user.is_superuser:
```

Comment:
Should probably use `getattr(self.request.user, "is_superuser", False)` instead.